web/settings.py

```
@@ -12,6 +13,11 @@
 
 env_file = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), ".env")
 
+SECRET_KEY = "django-insecure-5kyff0s@l_##j3jawec5@b%!^^e(j7v)ouj4b7q6kru#o#a)o3"
```

Ruff (0.8.2)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)